docs/manual/developer/04_style_guide.md

```diff
@@ -382,9 +389,10 @@ Shall be true if the system needs to be rebooted in order for the changes to tak
 
 #### `strategy`
 
-Should be one of the following values:
+Shall be one of the following values:
 
 * configure
+* combination
```

Comment:
Could you give an example of `combination`, please?

Reply:
I don't have an example off the top of my head. This was done to align the XCCDF 1.2 spec.